pom.xml

```diff
@@ -93,6 +92,24 @@
       <artifactId>docker-fixtures</artifactId>
       <version>1.0</version>
       <scope>test</scope>
+      <exclusions>
```

Comment:
[changelog](https://github.com/jenkinsci/docker-fixtures/#11-2017-oct-12)

Reply:
@jglick AFIU, you're proposing to upgrade the docker-fixtures version instead of exclude dependencies. Currently it is not possible due to JavaContainer can't be built.
I've raised this [JIRA issue](https://issues.jenkins-ci.org/browse/JENKINS-48310) and commited this [PR](https://github.com/jenkinsci/docker-fixtures/pull/11) to fix it. I think a new version will have to be released.